codespell_lib/data/dictionary_code.txt

```diff
@@ -6,6 +6,7 @@ cas->case, cast,
 clas->class
 cloneable->clonable
 cmo->com
+commitish->committish
```

Comment:
When is `commitish` valid in a computing context? In my quick searches I see `committish` as the preferred spelling, or `commit-ish`, but not `commitish`.